include/fastdds/dds/xtypes/dynamic_types/DynamicTypeBuilder.hpp

```diff
@@ -0,0 +1,205 @@
+     * @retval RETCODE_OK always.
+     */
+    virtual ReturnCode_t get_all_members_by_name(
+            DynamicTypeMembersByName& member) = 0;
+    /*!
+     * Returns a member looked for by @ref MemberId.
+     * @param[inout] member @ref DynamicTypeMember reference used to return the member.
+     * @param[in] id Member identifier.
+     * @return @ref ReturnCode_t
+     * @retval RETCODE_OK when member was found.
+     * @retval RETCODE_BAD_PARAMETER when member wasn't found.
+     */
+    virtual ReturnCode_t get_member(
+            traits<DynamicTypeMember>::ref_type member,
+            MemberId id) = 0;
+
+    /*!
+     * Returns all members sorted by MemberId.
+     * @param[inout] member @ref DynamicTypeMemberById reference used to return all members.
+     * @return @ref ReturnCode_t
+     * @retval RETCODE_OK always.
+     */
+    virtual ReturnCode_t get_all_members(
+            DynamicTypeMembersById& member) = 0;
+
+    /*!
+     * This operation returns the current number of members.
+     * @return Current number of members
+     */
+    virtual uint32_t get_member_count() = 0;
+
+    /*!
+     * This operation returns the member that corresponds to the specified index.
+     * @param[inout] member @ref DynamicTypeMember reference used to return the member.
+     * @param[in] index Index
+     * @return @ref ReturnCode_t
+     * @retval RETCODE_OK when member was found.
+     * @retval RETCODE_BAD_PARAMETER when index is out-of-range.
+     */
+    virtual ReturnCode_t get_member_by_index(
+            traits<DynamicTypeMember>::ref_type member,
+            uint32_t index) = 0;
+
+    /*!
+     * This operation returns the current number of annotations to the type.
+     * @return Current number of annotations
+     */
+    virtual uint32_t get_annotation_count() = 0;
+
+    /*!
+     * This operation returns the annotation that corresponds to the specified index.
+     * @param[inout] descriptor @ref AnnotationDescriptor reference where information is copied.
+     * @param[in] idx Index
+     * @return @ref ReturnCode_t
+     * @retval RETCODE_OK when member was found.
+     * @retval RETCODE_BAD_PARAMETER when reference is nil or index is out-of-range.
+     */
+    virtual ReturnCode_t get_annotation(
+            traits<AnnotationDescriptor>::ref_type descriptor,
+            uint32_t idx) = 0;
+
+    /*!
+     * Compares current state against a @ref DynamicType reference.
+     * @param[in] other @ref DynamicType reference to compare to.
+     * @return \b bool `true` on equality
+     */
+    virtual bool equals(
+            traits<DynamicType>::ref_type other) = 0;
+
+    /*!
+     * Adds a 'member' to this type, where the new 'member' has the meaning defined in the specification of
+     * the DynamicTypeMember class.
+     * @param[in] descriptor @ref MemberDescriptor reference used for the new member.
+     * @return ReturnCode_t
+     * @retval RETCODE_OK when the member was created successful.
```

Comment:
```suggestion
     * @retval RETCODE_OK when the member was created successfully.
```

Please apply this suggestion to other similar cases.

Reply:
Done in 44462498b